build.sh

```diff
@@ -46,4 +49,15 @@ build-auspice() {
     cd ..
 }
 
+build-forecasts() {
+    git clone git@github.com:nextstrain/forecasts-ncov.git
```

Comment:
Do we want this line to look like the following instead?

```
git clone https://github.com/nextstrain/forecasts-ncov.git
```

[CI fails to clone this repo due to a public key error](https://github.com/nextstrain/nextstrain.org/actions/runs/3746695720/jobs/6362237409#step:6:395) and I get the same error from local command line.

Reply:
Thanks John! Implemented this change and both CI + heroku can now build.

(I'm going to change to something like `git subtree`, but that's separate.)